lemminx-liberty/src/test/java/io/openliberty/LibertyCompletionTest.java

```diff
@@ -264,4 +296,78 @@ public void testFeatureRepetitionCompletionItem() throws BadLocationException {
+                CompletionItem httpsCompletion = c("${default.https.port}", "${default.https.port}");
+                final int TOTAL_ITEMS = 2; // total number of available completion items
+
+                XMLAssert.testCompletionFor(serverXML, null, serverXMLURI, TOTAL_ITEMS, httpCompletion,
+                        httpsCompletion);
+
+                serverXML = String.join(newLine, //
+                        "<server description=\"Sample Liberty server\">", //
+                        "       <featureManager>", //
+                        "                <platform>javaee-6.0</platform>", //
+                        "                <feature>acmeCA-2.0</feature>", //
+                        "       </featureManager>", //
+                        " <httpEndpoint host=\"*\" httpPort=\"kkpp|\"\n",//
+                        "                  httpsPort=\"${default.https.port}\" id=\"defaultHttpEndpoint\"/>",//
+                        "</server>" //
+                );
+                XMLAssert.testCompletionFor(serverXML, null, serverXMLURI, 0);
+        }
+
+        @Test
+        public void testVariableCompletionItemWithDefaultXsdValues() throws BadLocationException {
+                String serverXML = String.join(newLine, //
+                        "<server description=\"Sample Liberty server\">", //
+                        "       <featureManager>", //
+                        "                <platform>javaee-6.0</platform>", //
+                        "                <feature>acmeCA-2.0</feature>", //
+                        "       </featureManager>", //
+                        " <applicationManager autoExpand=\"f|\"/>",//
+                        "</server>" //
+                );
+                Map<String,String> propsMap=new HashMap<>();
+                propsMap.put("default.http.port","9080");
+                propsMap.put("default.https.port","9443");
+                propsMap.put("testVar","false");
+                propsMap.put("testVar2","true");
+                Properties props = new Properties();
+                props.putAll(propsMap);
+
+                Map<String,Properties> variables=new HashMap<>();
+                variables.put(srcResourcesDir.toURI().toString(),props);
+                when(settingsService.getVariablesForServerXml(any())).thenReturn(props);
+                CompletionItem httpCompletion = c("${default.http.port}", "${default.http.port}");
+                CompletionItem httpsCompletion = c("${default.https.port}", "${default.https.port}");
+                final int TOTAL_ITEMS = 4; // total number of available completion items
+                // variables values -> default.http.port, default.https.port
+                // default schema provided value, this is not being filtered -> true, false
+                XMLAssert.testCompletionFor(serverXML, null, serverXMLURI, TOTAL_ITEMS, httpCompletion,
```

Comment:
The provided serverXML does not contain any reference to http ports. What is this completing?

Reply:
In line#350, we had " <applicationManager autoExpand=\"f|\"/>",//
This was being auto completed by the variables default.http.port and default.https.port

to avoid confusion, I have changed this to httpEndPoint